e2e/helpers/tables.ts

```diff
@@ -95,15 +100,17 @@ export const enterTableData = async (
   columnsSchemaConfig: ColumnSchemaConfig[],
   tableData: TableDataConfig[],
 ) => {
-  const tableRows = await getTableEditorRows(page)
+  const tableRows = getTableEditorRows(page)
 
   // skip initial empty column
   await expectTableColumnHeaders(tableRows, columnsSchemaConfig, 1)
 
   await test.step('enter test data', async () => {
     const addRowButton = page.getByRole('button', { name: 'Add Row' })
     for (let index = 0; index < tableData.length; index++) {
-      await test.step(`enter test data: ${tableData[index].item}`, async () => {
+      const item = tableData[index].item.toString()
```

Comment:
`item` can be a string, string[], or number, so convert to a string before using in the template expression